desktop/platform/index.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+/**
+ * Internal dependencies
+ */
+const os = require( 'os' );
```

Comment:
Does this have some hidden behavior? It doesn't get called anywhere else in this code…